client/config/toml.go

```diff
@@ -40,7 +39,7 @@ func writeConfigToFile(configFilePath string, config *ClientConfig) error {
 		return err
 	}
 
-	return ioutil.WriteFile(configFilePath, buffer.Bytes(), 0644)
+	return ioutil.WriteFile(configFilePath, buffer.Bytes(), 0600)
```

Comment:
linter output: client/config/toml.go:42:9: G306: Expect WriteFile permissions to be 0600 or less (gosec)

Reply:
Bullshit. Ignore it, please (hint: use `// nolint:gosec`)